### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (h3)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2313]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[MakeCode Arcade](http://localhost:3232/index.html)
#### Element path
h3
#### Snippet
`<h3 class="header">Not finding what you're looking for?</h3>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.97 (foreground color: #25a25a, background color: #e5f9e7, font size: 13.7pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

I think this applies to most of the 'hint' like things, this one is the little green message at the bottom of the github import screen
